lib/storage_tables.rb

```diff
@@ -32,6 +32,8 @@ module StorageTables
 
   mattr_accessor :queues, default: {}
 
+  mattr_accessor :service_urls_expire_in, default: 5.minutes
```

Comment:
Hier misschien 15 mins van maken? Is net zo veilig, maar dan heb je misschien minder kans dat debiteuren klagen dat ze het document niet meer kunnen lezen.

```suggestion
  mattr_accessor :service_urls_expire_in, default: 15.minutes
```